README.md

```diff
@@ -7,7 +7,7 @@ The default stack in this repo includes three modular components:
 
 | Component                  | Description                                                                                                                                                           | Why it's useful                                                                                                                                                                         |
 |----------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| ML Code                    | Example ML project structure, with unit tested Python modules and notebooks using [MLflow pipelines](https://mlflow.org/docs/latest/pipelines.html)                   | Quickly iterate on ML problems, without worrying about refactoring your code into tested modules for productionization later on.                                                        |
+| ML Code                    | Example ML project structure, with unit tested Python modules and notebooks using [MLflow recipes](https://mlflow.org/docs/latest/pipelines.html)                     | Quickly iterate on ML problems, without worrying about refactoring your code into tested modules for productionization later on.                                                        |
```

Comment:
We can probably change the link `https://mlflow.org/docs/latest/pipelines.html` already since I am assuming that before merging we could have the new URLs deployed already. What do you think?
This implies to all the links in this PR

Reply:
The current PR uses pip to install a commit version from github. We can change the link and the requirement files together once it's released.

Reply:
I'd suggest to make it self-consistent with mlflow 2.0 release. Tracking these small changes are time consuming and doesn't worth that much given the release is only 1 week away.